src/site/content/en/blog/usb/index.md

```diff
@@ -119,7 +119,7 @@ are simply ECMAScript 2015 [Arrow functions].
 
 You can either prompt the user to select a single connected USB device using
 `navigator.usb.requestDevice()` or call `navigator.usb.getDevices()` to get a
-list of all connected USB devices the origin has access to.
+list of all connected USB devices the website has been granted access to.
```

Comment:
'Website' is less specific than 'origin'. Is this really what you want it to say?

Reply:
In that context, I think it's okay to say that. That's a pattern I've used in other Device APIs. See https://web.dev/serial/#:~:text=returns%20a%20list%20of%20serial%20ports%20the%20website%20has%20been%20granted%20access%20to.